.github/workflows/unit-tests.yml

```diff
@@ -15,59 +15,60 @@ on:
   push:
     branches:
       - '**'
-
-env:
-  node_version: 14.16.0
+  pull_request:
+    types: [opened, synchronize, reopened]
 
 jobs:
-  scala-test:
+  unit-test:
     strategy:
       matrix:
-        os: [macos-11, ubuntu-20.04]
+        os: [windows-latest, macos-latest, ubuntu-latest]
     name: Build ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
     steps:
-      - uses: actions/checkout@v2.3.5
-        with:
-          submodules: recursive
-      - uses: actions/setup-java@v2.5.0
-        with:
-          distribution: 'adopt'
-          java-version: '11'
-      - name: Run tests 📋
-        run: sbt test
-      - name: Checkout 🛎️
-        uses: actions/checkout@v2
+      - name: Enable Developer Command Prompt 💻
+        if: runner.os == 'Windows'
+        uses: ilammy/msvc-dev-cmd@v1
+      - name: Setup Java ☕
+        uses: actions/setup-java@v2
         with:
-          submodules: recursive
-      - name: Set up Python 🐍
+          distribution: adopt
+          java-version: 8
+      - name: Setup Python 🐍
         uses: actions/setup-python@v2
         with:
-          python-version: 3.8
+          python-version: '3.x'
       - name: Install Conan 💪
-        run: pip install conan pytest && conan --version
+        uses: turtlebrowser/get-conan@main
+      - name: Setup cmake 🔧
+        uses: lukka/get-cmake@latest
```

Comment:
Can this use `jwlawson/actions-setup-cmake@v1` instead? Or switch to this one in `.github/workflows/scala.yml` so that we are consistent between files.

Reply:
Lukka gets Ninja and cmake.   Turtlebrowser I copied over from the Scala workflow from John.